tests/nft/nft_linked_addresses_assert.sql

```diff
@@ -0,0 +1,18 @@
+-- Check against manually selected seed data
```

Comment:
just an fyi for future PRs, we're looking internally at how we leverage seed files & tests and we're likely moving away from pulling data from postgres, but rather building seeds manually to fit specific test cases (could be column based, aggregate based, etc). we can leave this for now, but it may be revised in the future

Reply:
Thank you for the feedback. I'll study other tests about manual building seeds data.

Currently I mainly focusing on Ethereum data. I'll also check how to improve data quality for Solana part.